index_impl.go

```diff
@@ -605,6 +639,21 @@ func (i *indexImpl) SearchInContext(ctx context.Context, req *SearchRequest) (sr
 		ctx = context.WithValue(ctx, search.FieldTermSynonymMapKey, fts)
 	}
 
+	scoringModelCallback := func() string {
+		if isBM25Enabled(i.m) {
```

Comment:
Related to other commend, let's update this method to return the scoring model to use.
To establish if it's bm25, you can add the extra check - `scoringModel() == index.BM25Scoring`.

Reply:
Perhaps this is also fine as is.